Example/Tests/Calls/SwiftyMockReactiveCallsSpec.swift

```diff
@@ -0,0 +1,248 @@
+                            sut.sum.performs { _ in .failure(TestError(id: 1)) }
+                        }
+
+                        it("should use failure logic stub instead of failure value") {
+                            expect(sut.sum(left: 15, right: 12)).to(fail(with: TestError(id: 1)))
+                        }
+                    }
+                }
+
+                context("when calling filtered value stubbed method") {
+                    beforeEach {
+                        sut.sum.returns(.success(10))
+                        sut.sum.on { $0.left  == 12 }.returns(.success(0))
+                        sut.sum.on { $0.right == 15 }.returns(.success(7))
+                        sut.sum.on { $0.right == 42 }.returns(.failure(TestError()))
+                    }
+                    context("when parameters matching filter") {
+                        it("should return filter srubbed value") {
+                            expect(sut.sum(left: 12, right:  2)).to(sendValue(0))
+                            expect(sut.sum(left: 0,  right: 15)).to(sendValue(7))
+                            expect(sut.sum(left: 23, right: 42)).to(fail(with: TestError()))
+                        }
+                    }
+                    context("when parameters don't match filters") {
+                        it("should return default stubbed value") {
+                            expect(sut.sum(left: 13, right: 2)).to(sendValue(10))
+                        }
+                    }
+                }
+
+                context("when calling filtered block stubbed method") {
+                    beforeEach {
+                        sut.sum.performs { .success($0.left - $0.right) }
+                        sut.sum.on { $0.left  == 0  }.performs { _ in .success(0) }
+                        sut.sum.on { $0.right == 0  }.performs { _ in .success(12) }
+                        sut.sum.on { $0.right == -1 }.performs { _ in .failure(TestError()) }
+                    }
+                    context("when parameters matching filter") {
+                        it("should return call filter-based block") {
+                            expect(sut.sum(left: 0,  right:  2)).to(sendValue(0))
+                            expect(sut.sum(left: 15, right:  0)).to(sendValue(12))
+                            expect(sut.sum(left: 15, right: -1)).to(fail(with: TestError()))
+                        }
+                    }
+                    context("when parameters don't match filters") {
+                        it("should call default stubbed block") {
+                            expect(sut.sum(left: 13, right: 2)).to(sendValue(11))
+                        }
+                    }
+                }
+
+                context("when calling filtered stubbed with block method") {
+                    beforeEach {
+                        sut.sum.returns(.success(17))
+                        sut.sum
```

Comment:
here it's aligned into single line https://github.com/stanfy/SwiftyMock/pull/13/files#diff-d6965839e6151d26ca5863fbd9d1db7fR205, but there you decided you separate it into 3 lines, please unify formatting